### PR TITLE
Support worker_trace_events dataset in logspush jobs

### PR DESCRIPTION
**Pull request by dglsparsons**

### CODE REVIEW

**Comment by jacobbednarz** [comment]
pending this landing at https://developers.cloudflare.com/logs/reference/log-fields/

**Comment by tanushreesharma-cf** [comment]
Docs will be updated tomorrow

**Comment by jacobbednarz** [comment]
:heart: thanks @tanushreesharma-cf

File: .changelog/2025.txt [hunk 1]
```diff
@@ -0,0 +1,3 @@
+```release-note:enhancement
+Support "workers_trace_events" dataset parameter
```
Comment (jacobbednarz) [inline .changelog/2025.txt:2]:
```suggestion
resource/cloudflare_logpush_job: add support for `workers_trace_events`
```

**Pull request merged**